packages/expo-cli/src/commands/build-native/utils.ts

```diff
@@ -0,0 +1,69 @@
+import spawnAsync from '@expo/spawn-async';
+import { TurtleApi } from '@expo/xdl';
+import delayAsync from 'delay-async';
+import ora from 'ora';
+
+import log from '../../log';
+import { BuildInfo } from './Builder';
+
+const table: any = require('../utils/cli-table.js');
```

Comment:
Please rewrite `cli-table.js` to ts and change the line to:
```suggestion
import table from '../utils/cli-table';
```